bgetech/integrationpluginbgetech.json

```diff
@@ -223,6 +223,167 @@
                             "defaultValue": 0.00
                         }
                     ]
+                },
+                {
+                    "name": "sdm72",
+                    "displayName": "SDM72",
+                    "id": "ce54ada6-cbe8-406a-9262-c707c4d8c392",
+                    "createMethods": ["discovery"],
+                    "interfaces": ["energymeter", "connectable"],
+                    "discoveryParamTypes": [
+                        {
+                            "id": "27219539-443b-43dc-ad8b-11e7cfb862b5",
+                            "name": "slaveAddress",
+                            "displayName": "Slave address",
+                            "type": "int",
+                            "defaultValue": 1
+                        }
+                    ],
+                    "paramTypes": [
+                        {
+                            "id": "6ec32145-f77c-4268-ae1d-159f9702fd85",
+                            "name": "slaveAddress",
+                            "displayName": "Modbus slave address",
+                            "type": "uint",
+                            "defaultValue": 1
+                        },
+                        {
+                            "id": "6edc9c2b-e939-4c24-ac81-fc3ca17aab3f",
+                            "name": "modbusMasterUuid",
+                            "displayName": "Modbus RTU master",
+                            "type": "QUuid",
+                            "defaultValue": "",
+                            "readOnly": true
+                        }
+                    ],
+                    "stateTypes": [
+                        {
+                            "id": "3b69f8e9-1788-4b71-8bd6-bece89f1d045",
+                            "name": "connected",
+                            "displayName": "Connected",
+                            "displayNameEvent": "Connected changed",
```

Comment:
all `displayNameEvent` properties are deprecated. The are not required any more.